### PR TITLE
Update Set-MpPreference.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated DisableNetworkProtectionPerfTelemetry, DisableSmtpParsing, IntelTDTEnabled and ThrottleForScheduledScanOnly

Fixes https://github.com/MicrosoftDocs/windows-powershell-docs/issues/3504

Fixes https://github.com/MicrosoftDocs/windows-powershell-docs/issues/3505

Fixes https://github.com/MicrosoftDocs/windows-powershell-docs/issues/3506

Fixes https://github.com/MicrosoftDocs/windows-powershell-docs/issues/3508

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by HotCakeX** [comment]
Thank you!

File: docset/winserver2022-ps/defender/Set-MpPreference.md [hunk 1]
```diff
@@ -613,6 +615,24 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -DisableNetworkProtectionPerfTelemetry 
+This setting disables the gathering and send of performance telemetry from Network Protection. 
+The accepted values are 0 and 1
+1- 	Network protection telemetry is disabled.
+0 (Default)- 	Network protection telemetry is enabled.
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/defender/Set-MpPreference.md:622]:
```suggestion
This setting disables the gathering and sending of performance telemetry from network protection. 
The accepted values are 0 and 1.
- 1- Network protection telemetry is disabled.
- 0 (Default) - Network protection telemetry is enabled.
```

File: docset/winserver2022-ps/defender/Set-MpPreference.md [hunk 2]
```diff
@@ -754,6 +774,24 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -DisableSmtpParsing 
+This setting disables SMTP Parsing for Network Protection.
+The accepted values are 0 and 1
+1- 		SMTP parsing is disabled.
+0 (Default)- 	SMTP parsing is enabled.
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/defender/Set-MpPreference.md:781]:
```suggestion
This setting disables SMTP parsing for network protection.
The accepted values are 0 and 1.
- 1 - SMTP parsing is disabled.
- 0 (Default) - SMTP parsing is enabled.
```

File: docset/winserver2022-ps/defender/Set-MpPreference.md [hunk 3]
```diff
@@ -1002,6 +1040,27 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -IntelTDTEnabled
+This policy setting configures the Intel TDT integration level for Intel TDT-capable devices.
+The acceptable values for this parameter are:
+
+- 0 (Default) - If you don't configure this setting, the default value will be applied. The default value is controlled by Microsoft security intelligence updates. Microsoft will enable Intel TDT if there is a known threat.
+- 1 - If you configure this setting to enabled, Intel TDT integration will turn on.
+- 2 - If you configure this setting to disabled, Intel TDT integration will turn off.
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/defender/Set-MpPreference.md:1049]:
```suggestion
- 0 (Default) - If you don't configure this setting, the default value will be applied. The default value is controlled by Microsoft security intelligence updates. Microsoft will enable Intel TDT if there is a known threat.
- 1 - If you configure this setting to enabled, Intel TDT integration will turn on.
- 2 - If you configure this setting to disabled, Intel TDT integration will turn off.
```

File: docset/winserver2022-ps/defender/Set-MpPreference.md [hunk 4]
```diff
@@ -1884,6 +1943,27 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -ThrottleForScheduledScanOnly
+A CPU usage limit can be applied to scheduled scans only, or to scheduled and custom scans. The default value applies a CPU usage limit to scheduled scans only.
+The acceptable values for this parameter are:
+
+1 (Default) - If you enable this setting, CPU throttling will apply only to scheduled scans.
+0- If you disable this setting, CPU throttling will apply to scheduled and custom scans.
+
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/defender/Set-MpPreference.md:1952]:
```suggestion
- 1 (Default) - If you enable this setting, CPU throttling will apply only to scheduled scans.
- 0 - If you disable this setting, CPU throttling will apply to scheduled and custom scans.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Comment by scanum** [comment]
@tiburd 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**